hooks/src/styra_opa_hook/handlers.py

```diff
@@ -74,7 +74,7 @@ def opa_query(
         headers = {"Authorization": f"Bearer {token}"}
 
     try:
-        resp = requests.post(type_configuration.opaUrl, json=opa_input, headers=headers)
+        resp = requests.post(type_configuration.opaUrl, json=opa_input, headers=headers, timeout=10)
```

Comment:
Should we `except requests.exceptions.ConnectTimeout` below and log something accordingly?

Reply:
Certainly could! Maybe something like?

```
 LOG.error("Timeout connecting to OPA at %s in %s seconds", type_configuration.opaUrl, connect_timeout)
```

Reply:
Sounds good to me!

Reply:
Let me know how that looks